app/helpers/calendar_helper.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module CalendarHelper
+
+  def product_calendar_view(product)
+    return "month" if product.daily_booking?
+
+    "agendaWeek"
```

Comment:
nit: in this cases I usually go for a regular if-else or even a ternary. When it's just to return a value, I find any of them more readable than this pattern